modules/nw-osp-configuring-external-load-balancer.adoc

```diff
@@ -15,23 +48,38 @@ on {rh-openstack-first}
 endif::[]
 to use an external load balancer in place of the default load balancer.
 
-// Maybe an About mod in support
+ifdef::vsphere,vmc[]
+You can also configure an {product-title} cluster to use an external load balancer that supports multiple subnets. You must configure the external load balancer with targets for port 6443. Each target must point to control plane nodes in each subnet where you provisioned the control plane nodes. 
 
+You do not need to define API and Ingress VIPs during installation. However, you must define the API and Ingress DNS A records, and then point these records to the external load balancer.
 
+You can reduce the complexity of maintaining external load balancer targets for the Ingress VIP by scheduling Ingress pods on control plane or infrastructure nodes.
+
+[NOTE]
+====
+Consider deploying control plane nodes and adding an Ingress machine set on networks that support light network traffic. This environment reduces the requirement to place a large number of IP address in the load balancer configuration file. 
+====
+endif::vsphere,vmc[]
+
+The example in the procedure demonstrates configuring an external load balancer for a cluster deployed in a single subnet.
 
 .Prerequisites
 
-* On your load balancer, TCP over ports 6443, 443, and 80 must be available to any users of your system.
+* On your load balancer, TCP over ports 22623, 6443, 443, and 80 must be reachable by all nodes in your cluster.
 
 * Load balance the API port, 6443, between each of the control plane nodes.
 
-* Load balance the application ports, 443 and 80, between all of the compute nodes.
+* Load balance the application ports, 443 and 80, between all the compute nodes.
 
 * On your load balancer, port 22623, which is used to serve ignition startup configurations to nodes, is not exposed outside of the cluster.
 
-* Your load balancer must be able to access every machine in your cluster. Methods to allow this access include:
-** Attaching the load balancer to the cluster's machine subnet.
-** Attaching floating IP addresses to machines that use the load balancer.
+* Your load balancer can access the required ports on each node in your cluster. You can ensure this level of access by completing the following actions: 
+** The API load balancer can access ports 22623 and 6443 on the control plane nodes.
+** The ingress load balancer can access ports 443 and 80 on the nodes where the ingress pods are located.
```

Comment:
I think the original wording on this is correct as well. we state the list of ports at the start of the prerequisites section so I don't think it needs to be reiterated here.

Reply:
This bullet point and its sub bullets detail LB + ports relationship. It goes into a bit more detail that bullet point 1. Perhaps I could mention API or Ingress LB in each of the subsequent bullet points?